plans/init.pp

```diff
@@ -1,33 +1,33 @@
 # Reboots targets and waits for them to be available again.
 #
-# @param nodes Targets to reboot.
+# @param targets Targets to reboot.
 # @param message Message to log with the reboot (for platforms that support it).
 # @param reboot_delay How long (in seconds) to wait before rebooting. Defaults to 1.
 # @param disconnect_wait How long (in seconds) to wait before checking whether the server has rebooted. Defaults to 10.
 # @param reconnect_timeout How long (in seconds) to attempt to reconnect before giving up. Defaults to 180.
 # @param retry_interval How long (in seconds) to wait between retries. Defaults to 1.
 # @param fail_plan_on_errors Raise an error if any targets do not successfully reboot. Defaults to true.
 plan reboot (
-  TargetSpec $nodes,
+  TargetSpec $targets,
   Optional[String] $message = undef,
   Integer[1] $reboot_delay = 1,
   Integer[0] $disconnect_wait = 10,
   Integer[0] $reconnect_timeout = 180,
   Integer[0] $retry_interval = 1,
   Boolean    $fail_plan_on_errors = true,
 ) {
-  $targets = get_targets($nodes)
+  $targs = get_targets($targets)
```

Comment:
@beechtom Sorry we should probably also use `target_objects` here

Reply:
Updated! 👍